frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -94,10 +130,13 @@ export default class ArenaClarificationList extends Vue {
   @Prop() problemAlias!: null | string;
   @Prop() username!: null | string;
   @Prop({ default: false }) showNewClarificationPopup!: boolean;
+  @Prop({ default: false }) filterAssignments!: boolean;
 
   T = T;
   PopupDisplayed = PopupDisplayed;
   currentPopupDisplayed = this.popupDisplayed;
+  selectedAssignment = null;
+  selectedProblem = null;
```

Comment:
```suggestion
  selectedAssignment: string | null = null;
  selectedProblem: string | null = null;
```

cuando se asignen cosas a `null`, hay que agregar el tipo completo.